libobs-d3d11/d3d11-texture2d.cpp

```diff
@@ -313,15 +313,24 @@ gs_texture_2d::gs_texture_2d(gs_device_t *device, ID3D11Texture2D *nv12tex,
 		InitRenderTargets();
 }
 
-gs_texture_2d::gs_texture_2d(gs_device_t *device, uint32_t handle)
+gs_texture_2d::gs_texture_2d(gs_device_t *device, uint32_t handle,
+			     bool ntHandle)
```

Comment:
Maybe an enum would be better than a bool, to support future types of handles?

Reply:
To be fair it probably doesn't matter, as this is an internal structure and not exposed as API.

Reply:
As this is an internal function, the API can change in the future if this needs to be expanded, without introducing any kind of breaking change. I do not suspect there to be any new kinds of handles in the future, so I personally am inclined to keep it as is here.